src/App.tsx

```diff
@@ -1,14 +1,52 @@
-import React from 'react';
+import { useCallback, useMemo, useState } from 'react';
+import { debounce } from 'lodash';
```

Comment:
Fix this error, please 
![image](https://github.com/mate-academy/react_autocomplete/assets/99403198/4426bf44-4fc1-4cb2-b31e-b469feef1086)